docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -1030,7 +1027,60 @@ See <<native-transport>> for details.
 
 IMPORTANT: Make sure your application has the right permissions to write to the socket.
 
-== Read only deployment environments
+[#use-io_uring]
+== Use io_uring
+
+IMPORTANT: `io_uring` is not support in native executables
+
+NOTE: `io_uring` support is experimental
+
+`io_uring` is a Linux kernel interface that allows you to send and receive data asynchronously.
+It provides unified semantics for both file and network I/O.
```

Comment:
@cescoffier this is true but we are not providing the vertx async file using this (while we should in a future, actually :P if matter!)

Reply:
Yes, here, it's mostly the theory, not the netty/vert.x implementation.

Reply:
Adding a note about the FS impl.

Reply:
Added.